### DISCUSSION TITLE
Consider providing source tarballs without assets

### DISCUSSION THREAD

**Discussion by jpalus** [ankitpokhrel/jira-cli#705] [Ideas]
As of this writing release tarballs are provided only for binary artifacts. The only way for distributions to fetch sources is to download automatic tarballs provided by github which include entire repository. That however results in a whopping sized tarballs of 51MB. Caused by 53MB taken by few gifs in `.github/assets` and 1MB by the actual source.

Would you please consider providing official source tarballs as a part of release that excludes those assets?

**Comment by ankitpokhrel** [comment]
Hi @jpalus, I think this is achievable by using [.gitattributes](https://git-scm.com/docs/gitattributes#_creating_an_archive) to ignore unnecessary files ([example](https://github.com/ankitpokhrel/tus-php/blob/main/.gitattributes) from another project). I will experiment with this on the next release. Thanks!
